go/vt/vtgate/planbuilder/testdata/info_schema57_cases.json

```diff
@@ -191,29 +191,28 @@
       "Instructions": {
         "OperatorType": "Distinct",
         "Collations": [
-          "(0:21)",
-          "(1:22)",
-          "(2:23)",
-          "(3:24)",
-          "(4:25)",
-          "5: binary",
-          "(6:26)",
-          "7: binary",
-          "8: binary",
-          "9: binary",
-          "10: binary",
-          "11: binary",
-          "12: binary",
-          "13: binary",
-          "(14:27)",
-          "(15:28)",
-          "(16:29)",
-          "(17:30)",
-          "18: binary",
-          "(19:31)",
-          "(20:32)"
+          "0: utf8mb4_0900_ai_ci",
```

Comment:
this is how happy this makes me
![very happy](https://media.giphy.com/media/vFKqnCdLPNOKc/giphy.gif)